apps/parsley/src/pages/index.tsx

```diff
@@ -26,6 +29,17 @@ const Content: React.FC = () => {
   localStorage.setItem("userId", user?.userId ?? "");
   useAnalyticAttributes(user?.userId ?? "");
 
+  const { pathname, search } = useLocation();
+  const { betaFeatures } = useMergedBetaFeatures();
+  useEffect(() => {
+    // If Parsley AI beta is enabled globally, and the user is opted in, redirect to the
+    // Parsley Beta URL. Limited to production as beta can only mirror production data.
+    if (isProduction() && betaFeatures?.parsleyAIEnabled) {
```

Comment:
I saw [this comment](https://github.com/evergreen-ci/ui/pull/1016#discussion_r2325733461) but was wondering if you considered implementing this as more of a backup? i.e. update Spruce links to also point to beta, and keep this so that if a user gets linked from somewhere else, then they're redirected. That may be less disruptive.